2024-04-09 17:46:18,828 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-09 17:46:18,829 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Configure stats pid to 7982
2024-04-09 17:46:18,830 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from /home/russellsb/.config/wandb/settings
2024-04-09 17:46:18,831 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/settings
2024-04-09 17:46:18,831 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-09 17:46:18,832 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-09 17:46:18,833 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-09 17:46:18,833 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-09 17:46:18,834 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-09 17:46:18,835 INFO    MainThread:7982 [wandb_init.py:_log_setup():521] Logging user logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240409_174618-s6jywf2n/logs/debug.log
2024-04-09 17:46:18,836 INFO    MainThread:7982 [wandb_init.py:_log_setup():522] Logging internal logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240409_174618-s6jywf2n/logs/debug-internal.log
2024-04-09 17:46:18,837 INFO    MainThread:7982 [wandb_init.py:_jupyter_setup():467] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f29206c1990>
2024-04-09 17:46:18,838 INFO    MainThread:7982 [wandb_init.py:init():561] calling init triggers
2024-04-09 17:46:18,838 INFO    MainThread:7982 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-09 17:46:18,839 INFO    MainThread:7982 [wandb_init.py:init():586] re-initializing run, found existing run on stack: 29t6xpdu
2024-04-09 17:46:18,841 INFO    MainThread:7982 [wandb_run.py:_finish():2064] finishing run russellsb/x-ray-chest-disease-classification/29t6xpdu
2024-04-09 17:46:18,842 INFO    MainThread:7982 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-09 17:46:18,844 INFO    MainThread:7982 [wandb_run.py:_restore():2287] restore
2024-04-09 17:46:18,844 INFO    MainThread:7982 [wandb_run.py:_restore():2293] restore done
2024-04-09 17:46:25,293 INFO    MainThread:7982 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-09 17:46:25,294 INFO    MainThread:7982 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-09 17:46:25,308 INFO    MainThread:7982 [wandb_run.py:_footer_sync_info():3895] logging synced files
2024-04-09 17:46:25,393 INFO    MainThread:7982 [wandb_init.py:init():611] starting backend
2024-04-09 17:46:25,393 INFO    MainThread:7982 [wandb_init.py:init():615] setting up manager
2024-04-09 17:46:25,398 INFO    MainThread:7982 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-04-09 17:46:25,399 INFO    MainThread:7982 [wandb_init.py:init():623] backend started and connected
2024-04-09 17:46:25,415 INFO    MainThread:7982 [wandb_run.py:_label_probe_notebook():1299] probe notebook
2024-04-09 17:46:25,416 INFO    MainThread:7982 [wandb_run.py:_label_probe_notebook():1309] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-04-09 17:46:25,416 INFO    MainThread:7982 [wandb_init.py:init():715] updated telemetry
2024-04-09 17:46:25,437 INFO    MainThread:7982 [wandb_init.py:init():748] communicating run to backend with 90.0 second timeout
2024-04-09 17:46:26,804 INFO    MainThread:7982 [wandb_run.py:_on_init():2357] communicating current version
2024-04-09 17:46:26,936 INFO    MainThread:7982 [wandb_run.py:_on_init():2366] got version response 
2024-04-09 17:46:26,936 INFO    MainThread:7982 [wandb_init.py:init():799] starting run threads in backend
2024-04-09 17:46:27,103 INFO    MainThread:7982 [wandb_run.py:_console_start():2335] atexit reg
2024-04-09 17:46:27,104 INFO    MainThread:7982 [wandb_run.py:_redirect():2190] redirect: wrap_raw
2024-04-09 17:46:27,104 INFO    MainThread:7982 [wandb_run.py:_redirect():2255] Wrapping output streams.
2024-04-09 17:46:27,104 INFO    MainThread:7982 [wandb_run.py:_redirect():2280] Redirects installed.
2024-04-09 17:46:27,106 INFO    MainThread:7982 [wandb_init.py:init():842] run started, returning control to user process
2024-04-09 17:46:27,125 INFO    MainThread:7982 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-09 17:46:27,125 INFO    MainThread:7982 [wandb_init.py:_pause_backend():432] pausing backend
2024-04-09 17:47:41,368 INFO    MainThread:7982 [wandb_init.py:_resume_backend():437] resuming backend
2024-04-09 17:47:41,430 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Current SDK version is 0.16.6
2024-04-09 17:47:41,431 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Configure stats pid to 7982
2024-04-09 17:47:41,431 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from /home/russellsb/.config/wandb/settings
2024-04-09 17:47:41,432 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/settings
2024-04-09 17:47:41,433 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-04-09 17:47:41,433 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-04-09 17:47:41,434 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-09 17:47:41,435 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {'api_key': '***REDACTED***'}
2024-04-09 17:47:41,436 INFO    MainThread:7982 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-04-09 17:47:41,437 INFO    MainThread:7982 [wandb_init.py:_log_setup():521] Logging user logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240409_174741-frsh9x9j/logs/debug.log
2024-04-09 17:47:41,437 INFO    MainThread:7982 [wandb_init.py:_log_setup():522] Logging internal logs to /mnt/c/Users/rsamm/root/repos/Data Science for Business Package/4. Operations Department/wandb/run-20240409_174741-frsh9x9j/logs/debug-internal.log
2024-04-09 17:47:41,438 INFO    MainThread:7982 [wandb_init.py:init():561] calling init triggers
2024-04-09 17:47:41,439 INFO    MainThread:7982 [wandb_init.py:init():568] wandb.init called with sweep_config: {}
config: {}
2024-04-09 17:47:41,439 INFO    MainThread:7982 [wandb_init.py:init():586] re-initializing run, found existing run on stack: s6jywf2n
2024-04-09 17:47:41,442 INFO    MainThread:7982 [wandb_run.py:_finish():2064] finishing run russellsb/x-ray-chest-disease-classification/s6jywf2n
2024-04-09 17:47:41,443 ERROR   MainThread:7982 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-04-09 17:47:41,444 INFO    MainThread:7982 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-04-09 17:47:41,444 INFO    MainThread:7982 [wandb_init.py:_jupyter_teardown():449] cleaning up jupyter logic
2024-04-09 17:47:41,445 INFO    MainThread:7982 [wandb_run.py:_atexit_cleanup():2304] got exitcode: 0
2024-04-09 17:47:41,446 INFO    MainThread:7982 [wandb_run.py:_restore():2287] restore
2024-04-09 17:47:41,446 INFO    MainThread:7982 [wandb_run.py:_restore():2293] restore done
2024-04-09 17:47:45,017 INFO    MainThread:7982 [wandb_run.py:_footer_history_summary_info():3936] rendering history
2024-04-09 17:47:45,018 INFO    MainThread:7982 [wandb_run.py:_footer_history_summary_info():3968] rendering summary
2024-04-09 17:47:45,028 INFO    MainThread:7982 [wandb_run.py:_footer_sync_info():3895] logging synced files
